packages/@react-aria/checkbox/docs/useCheckboxGroup.mdx

```diff
@@ -196,6 +198,31 @@ function Example() {
 }
 ```
 
+### Description
+
+The `description` prop can be used to associate additional help text with a checkbox group.
+
+```tsx example
+<CheckboxGroup label="Favorite sports" description="Select your favorite sports.">
+  <Checkbox value="soccer">Soccer</Checkbox>
+  <Checkbox value="baseball">Baseball</Checkbox>
+  <Checkbox value="basketball">Basketball</Checkbox>
+</CheckboxGroup>
+```
+
+### Error message
+
+The `errorMessage` prop can be used to help the user fix a validation error. It should be combined with the `validationState` prop to
+semantically mark the checkbox group as invalid for assistive technologies.
+
+```tsx example
+<CheckboxGroup label="Favorite sports" errorMessage="Invalid selection of sports.">
```

Comment:
should you provide `validationState` here?

Reply:
I think that's better, just updated. We aren't doing this for other examples, but maybe we should update those as well.